test/modules/TestResultCoordinator/Settings.cs

```diff
@@ -166,7 +166,7 @@ static Settings Create()
                 configuration.GetValue<ushort>("UNMATCHED_RESULTS_MAX_SIZE", DefaultUnmatchedResultsMaxSize),
                 configuration.GetValue<string>("TEST_INFO"),
                 configuration.GetValue("testMode", TestMode.Connectivity),
-                configuration.GetValue("unmatchedResultTolerance", TimeSpan.FromMinutes(1)),
```

Comment:
Don't you want to apply this change specifically to direct methods? I was expecting the change to be here:
https://github.com/Azure/iotedge/blob/master/e2e_deployment_files/connectivity_deployment.template.json#L900

Reply:
I am not familiar though with how this `unmatchedResultTolerance` gets used. Can you elaborate how it is different from what I linked?